TAO/TAO_IDL/driver/drv_preproc.cpp

```diff
@@ -138,6 +140,8 @@ DRV_cpp_putarg (const char *str)
       throw Bailout ();
     }
 
+  const ACE_TCHAR *arg_to_add = 0;
+
   if (str && ACE_OS::strchr (str, ' ') && !ACE_OS::strchr (str, '"'))
```

Comment:
I know this isn't your code... but is this first part even right? What if my string is `abc "bob"` ... it will escape the `"`s but not the ` `, right?

Reply:
I didn't pay much mind to it, but now that you mention it, I don't know. I don't know why that and what I'm trying to fix is a problem at all since it should be passing in arguments like `execv` in an `argv` style instead of apparently treating the whole dang thing as a single command line string.

Reply:
I think the part you're discussing is something I updated not too long ago to handle paths with spaces.  It wasn't meant to cover every single possibility, but just the case where you have an argument that is a filesystem path (for example, when this is called from DRV_add_include_path) with a space that's not already quoted.